cherrypy/_cpnative_server.py

```diff
@@ -7,6 +7,7 @@
 import cheroot.server
 
 import cherrypy
+from cherrypy._cpcompat import tonative
```

Comment:
I consider `tonative` to be deprecated. I'd like to avoid using it if possible and instead use either text or bytes explicitly on all Pythons.